engine/table/src/main/java/io/deephaven/engine/table/impl/locations/impl/AbstractTableLocation.java

```diff
@@ -222,7 +222,7 @@ private BasicDataIndex getDataIndex() {
 
     @Override
     @Nullable
-    public final BasicDataIndex getDataIndex(@NotNull final String... columns) {
+    public BasicDataIndex getDataIndex(@NotNull final String... columns) {
```

Comment:
As mentioned in the implementation, the implementation should not need to override this to properly initialize itself. We should leave this final.